utils/display/display.go

```diff
@@ -58,11 +64,13 @@ func PrintList(input interface{}, details Details) string {
 	for _, o := range objs {
 		bytes, err := json.Marshal(o)
 		if err != nil {
+			fmt.Printf("err1: %v\n", err)
```

Comment:
what `err1` & `err2` stand for?

Reply:
Good call out, I honestly dont remember why the errors are numbered....I'll remove the numbers